### ISSUE TITLE
speed-type-text will download a new file every time.

### ISSUE THREAD

**Issue by c02y** [hagleitn/speed-type#1]
Since `speed-type-text` will download a file from www.gutenberg.org, and that file usually contains a lot of  paragraphs, it can be used for many times. After executing `speed-type-text` again, why not randomly pick paragraphs from the downloaded files instead downloading a new file?

**Comment by hagleitn** [comment]
That's what it should be doing. I've configured it to use 20 books from Gutenberg. Each invocation will randomly choose a book and paragraph. If the book isn't yet stored in "~/emacs.d/speed-type/xxx.txt" it will download and store it in that place. So, you should see 20 downloads at most spread out over your first few invocations, but no more. Does that make sense? Do you think it's still downloading too much?

**Comment by c02y** [comment]
Yes, it's too much, as I said, one file which contains a lot of paragraphs 
can be used for many times, when executing `speed-type-text`, you can 
randomly pickup paragraphs from the downloaded file, maybe you can define 
variable that let us choose whether or not to download each time or define 
a new function such as `speed-type-text-new` to download a new file to 
start `speed-type-text`.

**Comment by c02y** [comment]
Yes, it's too much, as I said, one file which contains a lot of paragraphs can be used for many times, when executing `speed-type-text`, you can randomly pickup paragraphs from the downloaded file, maybe you can define variable that let us choose whether or not to download each time or define a new function such as `speed-type-text-new` to download a new file to start `speed-type-text`.

**Comment by c02y** [comment]
Test for rich text in my mobile email client.

Yes, it's too much, as I said, one file which contains a lot of paragraphs 
can be used for many times, when executing `speed-type-text`, you can 
randomly pickup paragraphs from the downloaded file, maybe you can define 
variable that let us choose whether or not to download each time or define 
a new function such as `speed-type-text-new` to download a new file to 
start `speed-type-text`.
